### DISCUSSION TITLE
Dúvida sobre os pacotes patrocinados

### DISCUSSION THREAD

**Discussion by GuilhermePesoa** [amidaware/tacticalrmm#1501] [Q&A]
Olá pessoal
Espero que estejam bem

Atualmente tenho 450 ativos gerenciados pelo Tactical RMM (todos Windows).
Tenho intenção de virar patrocinador e poder usufruir do agente para linux e mac, porém estou em dúvida sobre qual pacote se adequa a minha necessidade.
Pretendo abranger a gestão para pelo menos mais 50 servidores linux.
Com isso, eu deveria virar patrocinador com o pacote 1 (para 200 agentes) ou com o pacote 4 (para 600 agentes)?
Pra mim não ficou muito claro se a quantidade de agentes de cada pacote se refere ao todo (linux, mac e windows) ou somente aos agentes beta (linux e mac).
Poderiam me ajudar com isso?

**Comment by silversword411** [comment]
The recommended level is [Tier 3](https://docs.tacticalrmm.com/sponsor/) because you will have a total of 450+50=500 total agents, so 

<img width="361" alt="2023-05-04_122533 - sponsor level" src="https://user-images.githubusercontent.com/13395348/236265974-38ad67ec-3471-43e9-ac93-f54286a8e55e.png">
